streaming-atomfeed/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine/git as clone
+WORKDIR /app
+
+# TODO: Look over fixes and create PRs as needed to remove this requirement
+RUN ["git", "clone", "-b", "db-url-port-fix", "https://github.com/pmanko/atomfeed.git"]
+RUN ["git", "clone", "-b", "isanteplus-fixes", "https://github.com/pmanko/openmrs-module-atomfeed.git"]
```

Comment:
This clone and build process is the main issue I have with this PR. Basically we are depending on a custom solution in your repo. This makes me wonder whether it better that the rest of changes related to orchestrating `streaming-atomfeed` pipeline through docker be moved to your fork as well?